<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактирование/добавление записей</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>274</width>
     <height>618</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_id">
        <property name="text">
         <string>id</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="id"/>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Сорт</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="sort"/>
      </item>
      <item row="9" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Степень обжарки</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QComboBox" name="degree">
        <item>
         <property name="text">
          <string>Светлая</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Темная</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Высшая</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="14" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Молотый/зерно</string>
        </property>
       </widget>
      </item>
      <item row="18" column="1">
       <widget class="QComboBox" name="ground">
        <item>
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="19" column="0" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Описание вкуса</string>
        </property>
       </widget>
      </item>
      <item row="22" column="1">
       <widget class="QPlainTextEdit" name="description"/>
      </item>
      <item row="23" column="1">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Цена</string>
        </property>
       </widget>
      </item>
      <item row="25" column="1">
       <widget class="QDoubleSpinBox" name="price">
        <property name="maximum">
         <double>9999999.990000000223517</double>
        </property>
       </widget>
      </item>
      <item row="27" column="0" colspan="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Объем упаковки</string>
        </property>
       </widget>
      </item>
      <item row="30" column="1">
       <widget class="QDoubleSpinBox" name="volume">
        <property name="maximum">
         <double>9999999999.989999771118164</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="done_btn">
        <property name="text">
         <string>Применить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="close_btn">
        <property name="text">
         <string>Отменить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
